<!-- -->
<launch>
   <node pkg="psaf_scenario" type="scenario_runner_path.py" name="scenario_runner_commonroad_path" output="screen">
    <param name="file" value="path.debugpath" />
    <param name="timeout" value="-1" />
    <param name="sample_cnt" value="100" />
    <param name="radius" value="5" />
    <param name="height" value="0" />
 </node>
</launch>